app/views/catalog/mastheads/_government_documents.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="masthead-container">
+  <div id="masthead" class="government-documents-masthead">
+    <h1>Government documents</h1>
+    <div>
+      <p>Stanford is a depository library for state, federal, UN, and EU government documents.</p>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
Should we get real language before merging this, or is there a ticket to make sure we do this before deploying?